pr_agent/algo/ai_handler.py

```diff
@@ -68,22 +71,29 @@ def deployment_id(self):
         """
         return get_settings().get("OPENAI.DEPLOYMENT_ID", None)
 
-    @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
-           tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
-    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
+    @retry(
+        exceptions=(APIError, TryAgain, AttributeError, RateLimitError),
+        tries=OPENAI_RETRIES,
+        delay=2,
+        backoff=2,
+        jitter=(1, 3),
+    )
```

Comment:
**Suggestion:** It's a good practice to avoid hardcoding values directly in the code. The hardcoded values `2` and `(1, 3)` in the `retry` decorator for `chat_completion` method could be moved to a configuration file or defined as constants at the top of the file. This would make it easier to adjust these values in the future if needed. [best practice]